README.md

```diff
@@ -18,47 +18,71 @@
 
-## Key Features
+* **Query with SQL:** SQL-like language that supports selection, aggregation, filtering, group by, order by, distinct queries on data.
 
-- A column-oriented database with various compression schemes such as Run Length, Fixed Bit Length
-- Pluggable indexing technologies - Sorted Index, Bitmap Index, Inverted Index, Star-Tree Index
-- Ability to optimize query/execution plan based on query and segment metadata
-- Near real time ingestion from Kafka and batch ingestion from Hadoop
-- SQL like language that supports _selection, aggregation, filtering, group by, order by, distinct_ queries on fact data
-- Support for multivalued fields
-- Horizontally scalable and fault tolerant 
+* **Multi-valued fields:** support for multi-valued fields, allowing you to query fields as comma separated values.
 
-Because of the design choices we made to achieve these goals, there are certain limitations present in Pinot:
+* **Cloud-native on Kubernetes**: Helm chart provides a horizontally scalable and fault-tolerant clustered deployment that is easy to manage using Kubernetes.
 
-- Pinot is not a replacement for database i.e it cannot be used as source of truth store, cannot mutate data 
-- While Pinot supports text search, its not a replacement for search engine i.e relevance is not supported
-- Query cannot span across multiple tables - Use Presto-Pinot connector to achieve joins and other features
+## When should I use Pinot?
 
-Pinot works very well for querying time series data with lots of Dimensions and Metrics. Example - Query (profile views, ad campaign performance, etc.) in an analytical fashion (who viewed this profile in the last weeks, how many ads were clicked per campaign). 
+Pinot is designed to execute real-time OLAP queries with low latency on massive amounts of data and events. In addition to real-time stream ingestion, Pinot also supports batch use cases with the same low latency guarantees. It is suited in contexts where fast analytics, such as aggregations, are needed on immutable data, possibly, with real-time data ingestion. Pinot works very well for querying time series data with lots of dimensions and metrics.
 
-## Instructions to build Pinot
+Example query:
+```SQL
+SELECT sum(clicks), sum(impressions) FROM AdAnalyticsTable
+  WHERE
+       ((daysSinceEpoch >= 17849 AND daysSinceEpoch <= 17856)) AND
+       accountId IN (123456789)
+  GROUP BY
+       daysSinceEpoch TOP 100
+```
+
+Pinot is not a replacement for database i.e it cannot be used as source of truth store, cannot mutate data. While Pinot [supports text search](https://docs.pinot.apache.org/basics/features/text-search-support), it's not a replacement for a search engine. Also, Pinot queries cannot span across multiple tables by default. You can use the [Presto-Pinot connector](https://prestosql.io/blog/2020/05/25/pinot-connector.html) to achieve table joins and other features.
```

Comment:
The doc for Presto-Pinot connector links to prestosql. 
Please use or add prestodb's link (https://prestodb.io/docs/current/connector/pinot.html), as referenced in Pinot Presto Helm: https://github.com/apache/incubator-pinot/blob/master/kubernetes/helm/presto/Chart.yaml#L29